pdart/pipeline/Stage.py

```diff
@@ -2,16 +2,23 @@
 import os
 import os.path
 import traceback
+from typing import Optional
 
 from pdart.pipeline.Directories import Directories
 from pdart.pipeline.MarkerFile import BasicMarkerFile
 
 
 class Stage(metaclass=abc.ABCMeta):
-    def __init__(self, dirs: Directories, proposal_id: int) -> None:
+    def __init__(
+        self,
+        dirs: Directories,
+        proposal_id: int,
+        selected_suffixes: Optional[bool] = False,
```

Comment:
The pipeline always handles the big list of suffixes, so `selected_suffixes` will always be `False` in the pipeline.  So `selected_suffixes` is an unnecessary parameter here.